docs/autotune_install.md

```diff
@@ -31,8 +31,30 @@ Autotune can be deployed to a supported Kubernetes cluster. We currently support
 
 ### Install kubectl and minikube
 
+**Install kubectl**
+
+**`curl -LO "https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/bin/linux/amd64/kubectl"
```

Comment:
User should be able to copy these commands and paste it. Please update the entire read me wherever commands are present

Reply:
As discussed, also update the deploy.sh with the latest usage options and output.